New community content section (#6470)
withastro/docs: src/content/docs/en/community-resources/content.mdx

---
title: Community Educational Content
description: Discover community-produced tutorials, guides, articles and videos to help you learn and build with Astro!

> i18nReady: false

Reviewer: Note that while `talks.mdx` is `i18nReady: true`, this one is false, in case that's not on purpose.

Author: Thanks, Yan, I'll change it!

> i18nReady: false

Author: ```suggestion
i18nReady: true
```

All links are external, and may be based on earlier versions of Astro. Be sure to check the date on any community content, and adapt to your project accordingly.

For more Astro educational content, check out [the Astro Blog](https://astro.build/blog/) or sign up for [the official Astro newsletter](https://astro.build/newsletter/signup) where we recap the best community content of each month.

## Courses and Tutorials 

Courses and tutorials are focused on teaching you new concepts, often with exercises or sample projects to build. These are a great way to learn new concepts and give you the tools you need for working on your own Astro project.

Check out the following courses and tutorials to learn more about Astro.

### Introductory Tutorials
  - [Astro Web Framework Crash Course](https://www.youtube.com/watch?v=e-hTm5VmofI) by freeCodeCamp
  - [Astro Crash Course in 20 minutes](https://www.youtube.com/watch?v=zrPVTf761OI) by Chris Pennington (full paid course: [Learn Astro](https://learnastro.dev/))
  - [Astro 3.0 Crash Course](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course: [Build Modern Websites with Astro](https://astrocourse.dev/))

>   - [Everything You Need to Know about Astro](https://www.youtube.com/watch?v=rRxuVOutmFQ) by @awesome

Author: ```suggestion
```

---
title: Community Educational Content
description: Discover community-produced tutorials, guides, articles and videos to help you learn and build with Astro!
i18nReady: false
---

import Badge from '~/components/Badge.astro';

There is so much more out there that can help you learn and build with Astro! Here is some educational content produced and maintained by the Astro community.

All links are external, and may be based on earlier versions of Astro. Be sure to check the date on any community content, and adapt to your project accordingly.

For more Astro educational content, check out [the Astro Blog](https://astro.build/blog/) or sign up for [the official Astro newsletter](https://astro.build/newsletter/signup) where we recap the best community content of each month.

## Courses and Tutorials 

Courses and tutorials are focused on teaching you new concepts, often with exercises or sample projects to build. These are a great way to learn new concepts and give you the tools you need for working on your own Astro project.

Check out the following courses and tutorials to learn more about Astro.

> ### Introductory Tutorials
>   - [Astro Web Framework Crash Course](https://www.youtube.com/watch?v=e-hTm5VmofI) by freeCodeCamp
>   - [Astro Crash Course in 20 minutes](https://www.youtube.com/watch?v=zrPVTf761OI) by Chris Pennington (full paid course: [Learn Astro](https://learnastro.dev/))
>   - [Astro 3.0 Crash Course](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course: [Build Modern Websites with Astro](https://astrocourse.dev/))
>   - [Astro Crash Course in 60 minutes](https://www.youtube.com/watch?v=NniT0vKyn-E) by @developedbyed
>   - [Understanding Astro (ebook)](https://www.ohansemmanuel.com/books/understanding-astro) by Ohans Emmanuel
> 
>   ### Video Tutorials
>   - [Astro Crash Course in 20 minutes!](https://www.youtube.com/watch?v=zrPVTf761OI)
>   - [Astro Quick Start Course: Build an SSR Blog](https://www.youtube.com/watch?v=XoIHKO6AkoM)
>   - [How I created a movie application with Astro 3.x](https://www.youtube.com/watch?v=Gvr4WhgfP0w)
>   - [You may not actually understand Content Collections](https://www.youtube.com/watch?v=kwqlP6Pe0SI)
>   - [Everything You Need to Know about Astro](https://www.youtube.com/watch?v=rRxuVOutmFQ)
>   - [Build a custom blog platform with Astro and Apprwrite](https://www.youtube.com/watch?v=OERqwLy_reA)
>   - [Astro JS Portfolio Crash Course](https://www.youtube.com/watch?v=TwWvNK0yHjI)
>   - [Build a full stack blog with Astro](https://egghead.io/courses/build-a-full-stack-blog-with-astro-7ffcf9ec)

Author: ```suggestion
### Introductory Tutorials
  - [Astro Web Framework Crash Course](https://www.youtube.com/watch?v=e-hTm5VmofI) by freeCodeCamp
  - [Astro Crash Course in 20 minutes](https://www.youtube.com/watch?v=zrPVTf761OI) by Chris Pennington (full paid course: [Learn Astro](https://learnastro.dev/))
  - [Astro 3.0 Crash Course](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course: [Build Modern Websites with Astro](https://astrocourse.dev/))
  - [Everything You Need to Know about Astro](https://www.youtube.com/watch?v=rRxuVOutmFQ) by @awesome
  - [Astro Crash Course in 60 minutes](https://www.youtube.com/watch?v=NniT0vKyn-E) by @developedbyed
  - [Understanding Astro (ebook)](https://www.ohansemmanuel.com/books/understanding-astro) by Ohans Emmanuel

  ### Video Tutorials
  - [Astro Quick Start Course: Build an SSR Blog](https://www.youtube.com/watch?v=XoIHKO6AkoM)
  - [How I created a movie application with Astro 3.x](https://www.youtube.com/watch?v=Gvr4WhgfP0w)
  - [You may not ACTUALLY understand Content Collections…](https://www.youtube.com/watch?v=Fcw4c3wzm7I)
  - [Build a custom blog platform with Astro and Apprwrite](https://www.youtube.com/watch?v=OERqwLy_reA)
  - [Astro JS Portfolio Crash Course](https://www.youtube.com/watch?v=TwWvNK0yHjI)
  - [Build a full stack blog with Astro](https://egghead.io/courses/build-a-full-stack-blog-with-astro-7ffcf9ec)
```

  - [Astro 3.0 Crash Course](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course: [Build Modern Websites with Astro](https://astrocourse.dev/))
  - [Astro Crash Course in 60 minutes](https://www.youtube.com/watch?v=NniT0vKyn-E) by @developedbyed
  - [Understanding Astro (ebook)](https://www.ohansemmanuel.com/books/understanding-astro) by Ohans Emmanuel

  ### Video Tutorials
  - [Astro Crash Course in 20 minutes!](https://www.youtube.com/watch?v=zrPVTf761OI)
  - [Astro Quick Start Course: Build an SSR Blog](https://www.youtube.com/watch?v=XoIHKO6AkoM)
  - [How I created a movie application with Astro 3.x](https://www.youtube.com/watch?v=Gvr4WhgfP0w)
  - [You may not actually understand Content Collections](https://www.youtube.com/watch?v=kwqlP6Pe0SI)
  - [Everything You Need to Know about Astro](https://www.youtube.com/watch?v=rRxuVOutmFQ)
  - [Build a custom blog platform with Astro and Apprwrite](https://www.youtube.com/watch?v=OERqwLy_reA)
  - [Astro JS Portfolio Crash Course](https://www.youtube.com/watch?v=TwWvNK0yHjI)
  - [Build a full stack blog with Astro](https://egghead.io/courses/build-a-full-stack-blog-with-astro-7ffcf9ec)

> 

Author: ```suggestion
```

---
title: Community Content
description: Discover community-produced guides, articles and videos to help you learn and build with Astro!
i18nReady: false
---

import Badge from '~/components/Badge.astro';
import { CardGrid, LinkCard } from '@astrojs/starlight/components';
import Card from '~/components/Landing/Card.astro'
import ListCard from '~/components/Landing/ListCard.astro'

> PLACEHOLDER TEXT: There's lots more content out there that can help you learn and build with Astro! Here is some content produced and maintained by the Astro community. All links are external and we don't guarantee anything here....

Reviewer 2: ```suggestion
PLACEHOLDER TEXT: There are many published resources that can help you learn and build with Astro! This page highlights some content that is produced and maintained by the Astro community. All links are external and we don't guarantee anything here....
```
Perhaps a beginning to rounding this out? Not sure what you had in mind but I hope this helps!

Author: Thanks for looking this over, but your clues are a) this is a "draft" PR and b) I have the words PLACEHOLDER TEXT in bold to literally say that these are not my intended words, but creating space as a visual marker.

Way too early for anyone to chime in yet!  :smile:

Reviewer 2: Right. My apologies!!
I was just trying to generate some ideas, but I will video my time for now. Nevertheless, everything looks awesome :)

:::tip[Add your own!]
Have you produced educational content about Astro?

> Open a PR adding a link to this page!

Reviewer 2: ```suggestion
Open a PR to add a link to this page!
```
Just a little refactor here?
Or, maybe, `which adds`?

Courses and tutorials are focused on teaching you new concepts, often with exercises or sample projects to build. These are a great way to learn new concepts and give you the tools you need for working on your own Astro project.

Check out the following courses and tutorials to learn more about Astro.

### Introductory Tutorials

>   - [Astro Web Framework Crash Course](https://www.youtube.com/watch?v=e-hTm5VmofI) on freeCodeCamp

Author: ```suggestion
  - [Astro Web Framework Crash Course](https://www.youtube.com/watch?v=e-hTm5VmofI) by freeCodeCamp
```

>   - [Astro Crash Course in 20 minutes](https://www.youtube.com/watch?v=zrPVTf761OI) by @CodingInPublic / Chris Pennington (full paid course:  [Learn Astro](https://learnastro.dev/))

Author: ```suggestion
  - [Astro Crash Course in 20 minutes](https://www.youtube.com/watch?v=zrPVTf761OI) by Chris Pennington (full paid course:  [Learn Astro](https://learnastro.dev/))
```

>   - [Astro Crash Course in 20 minutes](https://www.youtube.com/watch?v=zrPVTf761OI) by Chris Pennington (full paid course:  [Learn Astro](https://learnastro.dev/))
>   - [Astro 3.0 Crash Course](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course [Build Modern Websites with Astro](https://astrocourse.dev/))

Author: ```suggestion
  - [Astro Crash Course in 20 minutes](https://www.youtube.com/watch?v=zrPVTf761OI) by Chris Pennington (full paid course: [Learn Astro](https://learnastro.dev/))
  - [Astro 3.0 Crash Course](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course: [Build Modern Websites with Astro](https://astrocourse.dev/))
```

  - [Astro Web Framework Crash Course on freeCodeCamp](https://www.youtube.com/watch?v=e-hTm5VmofI)
  - [Learn Astro by Chris Pennington <span ><Badge variant="neutral">Paid</Badge></span>](https://learnastro.dev/) 
  - [Build Modern Websites with Astro by James Q Quick <span><Badge variant="neutral">Paid</Badge></span>](https://astrocourse.dev/) 
  - [Understanding Astro by Ohans Emmanuel](https://www.ohansemmanuel.com/books/understanding-astro)

>   </ListCard>

Reviewer 2: I am unsure what it is, but the lack of space between the paragraph and the card here is somewhat jarring. Is there something we can do about this?
<img width="763" alt="image" src="https://github.com/withastro/docs/assets/20650404/bcb937fa-2646-4637-a4ea-3ec68688a7fe">

>   - [Astro 3.0 Crash Course (View Transitions, Image, Content Collections, and More!)](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course [Build Modern Websites with Astro](https://astrocourse.dev/)

Author: ```suggestion
  - [Astro 3.0 Crash Course (View Transitions, Image, Content Collections, and More!)](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course [Build Modern Websites with Astro](https://astrocourse.dev/))
```

>   - [Astro 3.0 Crash Course (View Transitions, Image, Content Collections, and More!)](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course [Build Modern Websites with Astro](https://astrocourse.dev/))

Author: ```suggestion
  - [Astro 3.0 Crash Course](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course [Build Modern Websites with Astro](https://astrocourse.dev/))
```

  - [Everything You Need to Know about Astro](https://www.youtube.com/watch?v=rRxuVOutmFQ)
  - [Build a custom blog platform with Astro and Apprwrite](https://www.youtube.com/watch?v=OERqwLy_reA)
  - [Astro JS Portfolio Crash Course](https://www.youtube.com/watch?v=TwWvNK0yHjI)
  - [Build a full stack blog with Astro](https://egghead.io/courses/build-a-full-stack-blog-with-astro-7ffcf9ec)

## Recipes and Guides

See guided examples of adding features to your Astro project, submitted by our community members!

[Astro recipes](/en/recipes/) are typically short, focused how-to guides that walk a reader through completing a working example of a specific task. Recipes are a great way to add new features or behavior to your Astro project by following step-by-step instructions!

Other guides might explain concepts related to an area of content, such as using images or working with MDX.

:::tip[Add your own!]
Have you written a recipe for Astro? [Edit this page](https://github.com/withastro/docs/edit/main/src/content/docs/en/community-resources/content.mdx) and add your link in the appropriate section!
:::

> 

Author: ```suggestion
```

---
title: Community Educational Content
description: Discover community-produced tutorials, guides, articles and videos to help you learn and build with Astro!
i18nReady: false
---

import Badge from '~/components/Badge.astro';

There is so much more out there that can help you learn and build with Astro! Here is some educational content produced and maintained by the Astro community.

All links are external, and may be based on earlier versions of Astro. Be sure to check the date on any community content, and adapt to your project accordingly.

> For more Astro educational content, check out [the Astro Blog](https://astro.build/blog/) or sign up for [the official Astro newsletter](https://astro.build/newsletter) where we recap the best community content of each month.

Reviewer 3: ```suggestion
For more Astro educational content, check out [the Astro Blog](https://astro.build/blog/) or sign up for [the official Astro newsletter](https://astro.build/newsletter/signup) where we recap the best community content of each month.
```

## Courses and Tutorials 

Courses and tutorials are focused on teaching you new concepts, often with exercises or sample projects to build. These are a great way to learn new concepts and give you the tools you need for working on your own Astro project.

Check out the following courses and tutorials to learn more about Astro.

### Introductory Tutorials

>   - [Astro Web Framework Crash Course on freeCodeCamp](https://www.youtube.com/watch?v=e-hTm5VmofI)
>   - [Learn Astro by Chris Pennington](https://learnastro.dev/) 
>   - [Build Modern Websites with Astro by James Q Quick](https://astrocourse.dev/) 
>   - [Understanding Astro by Ohans Emmanuel](https://www.ohansemmanuel.com/books/understanding-astro)
>   - [Astro Crash Course in 60 minutes by @developedbyed](https://www.youtube.com/watch?v=NniT0vKyn-E)

Author: ```suggestion
  - [Astro Web Framework Crash Course](https://www.youtube.com/watch?v=e-hTm5VmofI) on freeCodeCamp
  - [Astro Crash Course in 20 minutes](https://www.youtube.com/watch?v=zrPVTf761OI) by @CodingInPublic / Chris Pennington (full paid course:  [Learn Astro](https://learnastro.dev/))
  - [Astro 3.0 Crash Course (View Transitions, Image, Content Collections, and More!)](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course [Build Modern Websites with Astro](https://astrocourse.dev/)
  - [Astro Crash Course in 60 minutes](https://www.youtube.com/watch?v=NniT0vKyn-E) by @developedbyed
  - [Understanding Astro (ebook)](https://www.ohansemmanuel.com/books/understanding-astro) by Ohans Emmanuel
```

  ### Video Tutorials
  - [Astro Crash Course in 20 minutes!](https://www.youtube.com/watch?v=zrPVTf761OI)
  - [Astro Quick Start Course: Build an SSR Blog](https://www.youtube.com/watch?v=XoIHKO6AkoM)
  - [How I created a movie application with Astro 3.x](https://www.youtube.com/watch?v=Gvr4WhgfP0w)
  - [You may not actually understand Content Collections](https://www.youtube.com/watch?v=kwqlP6Pe0SI)
  - [Everything You Need to Know about Astro](https://www.youtube.com/watch?v=rRxuVOutmFQ)
  - [Build a custom blog platform with Astro and Apprwrite](https://www.youtube.com/watch?v=OERqwLy_reA)
  - [Astro JS Portfolio Crash Course](https://www.youtube.com/watch?v=TwWvNK0yHjI)
  - [Build a full stack blog with Astro](https://egghead.io/courses/build-a-full-stack-blog-with-astro-7ffcf9ec)

## Recipes and Guides

See guided examples of adding features to your Astro project, submitted by our community members!

> [Astro recipes](https://contribute.docs.astro.build/guides/recipe-writing/#is-my-guide-a-recipe) are typically short, focused how-to guides that walk a reader through completing a working example of a specific task. Recipes are a great way to add new features or behavior to your Astro project by following step-by-step instructions!

Author: ```suggestion
[Astro recipes](/en/recipes/) are typically short, focused how-to guides that walk a reader through completing a working example of a specific task. Recipes are a great way to add new features or behavior to your Astro project by following step-by-step instructions!
```

### Authentication
- [Add Github OAuth with Lucia](https://lucia-auth.com/guidebook/github-oauth/astro)
- [Add username and password authentication with Lucia](https://lucia-auth.com/guidebook/sign-in-with-username-and-password/astro)
### RSS
- [Use a dynamic filename when importing images](https://vaihe.com/blog/astro/astro-dynamic-image-prop/)
- [Add an RSS feed to Astro using Storyblok's stories](https://straffesites.com/en/blog/rss-feed-astro-storyblok)
- [How to Send Any Data to Your RSS Feed? A Guide with Astro + RSS](https://aritraroy.live/tutorial/blogs/2023/how-to-send-any-data-to-rss-feed/)
### Images
- [Add blog post images to your Astro RSS Feed](https://webreaper.dev/posts/astro-rss-feed-blog-post-images/)
- [Generate Open Graph images for an Astro blog](https://techsquidtv.com/blog/generating-open-graph-images-for-astro/)
### Editor Tooling
- [Get VSCode, ESLint & Prettier working with Astro](https://patheticgeek.dev/blog/astro-prettier-eslint-vscode)
- [Integrate Prettier with Astro and Tailwind CSS](https://straffesites.com/en/blog/integrate-prettier-astro-tailwindcss)
### Animation

> - [Add animated page transitions with Swup](https://navillus.dev/blog/astro-plus-swup)

Reviewer 4: Swup is something View Transitions provide with lesser setup, that being said this blogpost is outdated, I tried their instructions with Astro Starter template (could not make it work, even the repo they provide to `degit` seems unavailable😔)

Author: Ah, true this *is* a pretty old article. OK, if too old to be relevant, then we'll remove it. Thank you for checking!

> ### Animation
> - [Add animated page transitions with Swup](https://navillus.dev/blog/astro-plus-swup)

Author: ```suggestion
```

---
title: Community Educational Content
description: Discover community-produced tutorials, guides, articles and videos to help you learn and build with Astro!
i18nReady: false
---

import Badge from '~/components/Badge.astro';
import { CardGrid, LinkCard } from '@astrojs/starlight/components';
import Card from '~/components/Landing/Card.astro'

> import ListCard from '~/components/Landing/ListCard.astro'

Author: Yes, the grid looked super imposing and drawn out, harder to actually read. I shared some previews, and people like a plain list better.
